conversion/nifti_write.py

```diff
@@ -97,7 +97,7 @@ def nifti_write(inImg, prefix= None):
                 bvec.append(1)
                 # bvecINijk= RAS2IJK @ SPACE2RAS @ mf @ np.matrix(bvec).T
                 # simplified below
-                bvecINijk= xfrm_nhdr.T @ mf @ np.matrix(bvec).T
+                bvecINijk= xfrm_nhdr.T @ mf @ np.array(bvec).T
```

Comment:
Can you tell the advantage of array over matrix?

Reply:
matrix is deprecated, and there is a long discussion why that is on the numpy website.  the "matrix" class is the same as array, but with a few extra overload functions that are no longer needed now that the '@' operator is available to do the same behavior.